### DISCUSSION TITLE
Add way to start process from only a peak matrix

### DISCUSSION THREAD

**Discussion by GregorySchwartz** [GreenleafLab/ArchR#575] [Feature Requests]
I have some data that is processed and presented as a peak matrix. It would be quite useful to use this peak matrix as the starting point in ArchR. Is it possible to add this feature, without needing the BAM / fragment file?

**Comment by rcorces** [comment]
This is a longer term goal for us. I agree that it would be useful but its quite difficult to actually implement since there are so many aspects of an ArchRProject that arent captured by a simple peak matrix. I do not see this feature being implemented in the near term but hopefully one day,
